src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.common;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNullReturnsTrue() {
+        boolean result = Utils.isAnyNull("TEST", 123456789, null);
```

Comment:
Most of the code is the same for the rest of the tests, and should therefore be refactored.